### ISSUE TITLE
Unescaped area name can break users page

### ISSUE THREAD

**Issue by jaragunde** [Igalia/phpreport#564]
If you create an area in *Data Management -> Areas* with a name that contains the `'` character, like "I don't know", it will later break the user edition page (*Data Management -> Users*. It will be empty and show this error in the browser console:

> Uncaught SyntaxError: missing ] after element list

The reason is that, at some point, we generate a piece of JS code with a list of areas like this:
```
data : [
    [2, 'Web'],[3, 'Systems'],[4, 'Management'],[5, 'I don't know'],]
```

Where we use `'` to enclose strings, and we don't do any escaping of the string so we are generating incorrect JavaScript.

**Comment by jaragunde** [comment]
Same problem in the project management screen.

**Comment by jaragunde** [comment]
Reviewed in PR #573:
```
commit 4b3c7bdcc83aacd9ef005411e899fa79da6186ca
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Fri Mar 18 14:56:40 2022 +0100

    [#564] Escape area names for use in JS code.
    
    We need to use json_encode to properly escape user-entered names in
    generated JS code, which might contain special characters.

**Comment by jaragunde** [comment]
Same problem in project evaluation report.

**Comment by jaragunde** [comment]
Merged in PR #577:
```
commit a4bf9d4258dd8b97182ec3267aea6755a8dd2efb
Author: Jacobo Aragunde Pérez <jaragunde@igalia.com>
Date:   Fri Apr 8 09:54:05 2022 +0200

    [#564] Escape area names in project evaluation report.
    
    We need to use json_encode to properly escape user-entered names in
    generated JS code, which might contain special characters.
```
